webrtc.html

```diff
@@ -4076,35 +4076,73 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
-              composed only of case-sensitive alphanumeric characters (a-z,
-              A-Z, 0-9) up to a maximum of 16 characters.</p>
+              <p>When this method is invoked, the User Agent must run the
+              following steps:</p>
+              <ol>
+                <li>
+                  <p>If the dictionary argument is present, and it has a
+                  streams member, let <var>streams</var> be that list of <code>
+                  <a>MediaStream</a></code> objects, or an empty list
+                  otherwise.</p>
+                </li>
+                <li>
+                  <p>If the dictionary argument is present, and it has a
+                  sendEncodings member, let <var>sendEncodings</var> be that
+                  list of <code><a>RTCRtpEncodingParameters</a></code> objects,
+                  or an empty list otherwise.</p>
+                </li>
+                <li>
+                  <p>If the first argument is a string, let it be
+                  <var>kind</var> and run the following steps:</p>
+                  <ol>
+                    <li>
+                      <p>If <var>kind</var> is not a legal <code>
+                      <a>MediaStreamTrack</a></code> <code>kind</code>, throw a
+                      <code>TypeError</code> and abort these, and all further
+                      steps.</p>
+                    </li>
+                    <li>
+                      <p>Let <var>track</var> be <code>null</code>.</p>
+                    </li>
+                  </ol>
+                </li>
+                <li>
+                  <p>If the first argument is a <code>
+                  <a>MediaStreamTrack</a></code>, let it be <var>track</var>
+                  and let <var>kind</var> be <var>track.kind</var>.</p>
+                </li>
+                <li>
+                  <p><a>Create an RTCRtpSender</a> with <var>track</var>,
+                  <var>streams</var> and <var>sendEncodings</var> and let
+                  <var>sender</var> be the result.</p>
+                  <p>If <var>sendEncodings</var> is set, then subsequent
+                  calls to <code>createOffer</code> will be configured to send
+                  with multiple RTP encodings as defined in <span data-jsep=
+                  "subsequent-offers">[[!JSEP]]</span>. When
+                  <code>setRemoteDescription</code> is called with a
+                  corresponding remote description that is able to receive
+                  multiple RTP encodings as defined in [[!JSEP]], the <code>
+                  <a>RTCRtpSender</a></code> may send multiple RTP encodings
+                  and the parameters retrieved via the transceiver&#8217;s
+                  <code>sender.getParameters()</code> will reflect the
+                  encodings negotiated.</p> <p>RID values passed into
+                  <code>init.sendEncodings</code> must be composed only of
```

Comment:
I think that must is confusing; it would be better to describe what the UA should do if it is not the case (I realize that's a bug in the original text, not in this PR, but if fixing it is easy, we should probably do it; if not, I'll open an other issue)